fob413/airtech
# 163976136 User signin

**What does this PR do?**
- User can sign into the platform

**Description of Task to be completed?**
- Signin a user to the platform
- Signin an admin to the platform
- Write tests

**How should this be manually tested?**
- Start the app by following the readme
- Hit the endpoint with the valid request object `/api/v1/signin`
- Check circleci

**Any background context you want to provide?**
NA

**What are the relevant pivotal tracker stories?**
- https://www.pivotaltracker.com/story/show/163976136
